client/cl_gang.lua

```diff
@@ -52,291 +61,270 @@ end)
     QBCore.Functions.TriggerCallback('qb-gangmenu:server:GetAccount', function(cb)
         local SocietyMenu = {
             {
-                header = "Balance: $" .. comma_valueGang(cb) .. " - " .. string.upper(PlayerGang.label),
-                isMenuHeader = true,
-                icon = "fa-solid fa-circle-info",
-            },
-            {
-                header = "Deposit",
+                title = "Deposit",
                 icon = "fa-solid fa-money-bill-transfer",
-                txt = "Deposit Money",
-                params = {
-                    event = "qb-gangmenu:client:SocietyDeposit",
-                    args = comma_valueGang(cb)
-                }
+                description = "Deposit Money",
+                event = "qb-gangmenu:client:SocietyDeposit",
+                args = comma_valueGang(cb)
             },
             {
-                header = "Withdraw",
+                title = "Withdraw",
                 icon = "fa-solid fa-money-bill-transfer",
-                txt = "Withdraw Money",
-                params = {
-                    event = "qb-gangmenu:client:SocietyWithdraw",
-                    args = comma_valueGang(cb)
-                }
+                description = "Withdraw Money",
+                event = "qb-gangmenu:client:SocietyWithdraw",
+                args = comma_valueGang(cb)
             },
             {
-                header = "Return",
+                title = "Return",
                 icon = "fa-solid fa-angle-left",
-                params = {
-                    event = "qb-gangmenu:client:OpenMenu",
-                }
-            },
+                event = "qb-gangmenu:client:OpenMenu"
+            }
         }
-        exports['qb-menu']:openMenu(SocietyMenu)
+
+        lib.registerContext({
+            id = 'open_gangSociety',
+            title = "Balance: $" .. comma_valueGang(cb) .. " - " .. string.upper(PlayerGang.label),
+            options = SocietyMenu
+        })
+        lib.showContext('open_gangSociety')
     end, PlayerGang.name)
 end)
 
 RegisterNetEvent('qb-gangmenu:client:SocietyDeposit', function(saldoattuale)
-    local deposit = exports['qb-input']:ShowInput({
-        header = "Deposit Money <br> Available Balance: $" .. saldoattuale,
-        submitText = "Confirm",
-        inputs = {
-            {
-                type = 'number',
-                isRequired = true,
-                name = 'amount',
-                text = 'Amount'
-            }
+    local deposit = lib.inputDialog("Deposit Money", {
+        {
+            type = "number",
+            label = "Available Balance",
+            disabled = true,
+            default = saldoattuale
+        },
+        {
+            type = "number",
+            label = "Amount"
         }
     })
-    if deposit then
-        if not deposit.amount then return end
-        TriggerServerEvent("qb-gangmenu:server:depositMoney", tonumber(deposit.amount))
+
+    if not deposit then
```

Comment:
Also check if deposit is an empty table or not, it's empty if nothing was entered